packages/experiments/src/components/PersonaCoin/PersonaCoin.types.ts

```diff
@@ -92,6 +98,15 @@ export interface IPersonaCoinProps
    * @defaultvalue 'white'
    */
   initialsColor?: string;
+
+  /**
+   * Optional callback used to set a ref to the component
+   */
+  componentRef?: IRefObject<IPersonaCoinViewProps>;
```

Comment:
Instead of explicitly including this, you can also extend `IBaseProps<TComponentInterface>`. Generally the component interface is a limited programmatic set (usually just for focus scenarios) rather than the component props. (I've made all of this part of the create-component template.) 

What do you think of requiring this by default? Is it confusing?

Reply:
I'm not really sure why `componentRef` would need be present on each component :) But maybe it's just a Fabric convention we'd like to enforce?

Does it also call the `componentRef` function?  It's fine for me to extend from it but if it's not in foundation to call it I should probably also use it, which i currently haven't done (yet)

Reply:
Similar to `BaseComponent`, `BaseState` automatically calls this reference for you I'm curious though what @dzearing thinks about having this be the default for all components. If we don't want people using `componentRef` in general, maybe it's better to exclude it by default (or at least don't expose it via props.)